packages/components/psammead-grid/src/index.stories.jsx

```diff
@@ -1455,3 +1462,7 @@ storiesOf('Components|Grid', module)
     },
     { notes, knobs: { escapeHTML: false } },
   );
+
+buildRTLSubstories(STORY_KIND, {
+  include: ['Example with Top story and regular promos'],
```

Comment:
Similarly here there are two levels of nesting, with an RTL folder within an RTL folder. The examples look good, though.

Reply:
Same as above, I suspect this is a HMR specific issue which wouldn't happen in production. If you think it isn't I can take a look into a way to prevent duplication/nesting of stories.